### PR TITLE
Tailored the Jetpack step message to the services selected

### PR DESCRIPTION
**Pull request by marcinbot**
Fixes the message appearing on the Jetpack connection step to include the selected services that require a Jetpack connection. No message appears if there are no relevant services selected.

### CODE REVIEW

**Review by jeffstieler — DISMISSED** [review]
My apologies for forgetting to discuss translation strategy for this.

Unfortunately, we can't concatenate features because that enforces left-to-right.

Let's refactor this to have fully written out strings that can be translated based on which features have been enabled.

File: includes/admin/class-wc-admin-setup-wizard.php [hunk 1]
```diff
@@ -1168,13 +1168,36 @@ class_exists( 'Jetpack' ) &&
...
+			$description = __( "Your store's almost ready. Connect to Jetpack for full access to live rates and discounted shipping labels.", 'woocommerce' );
+		}
 		?>
 		<form method="post" class="activate-jetpack">
 			<h1><?php esc_html_e( 'Connect your store to Jetpack', 'woocommerce' ); ?></h1>
+			<?php if ( $description ) { ?>
 			<p>
-				<?php // TODO: tailor this message to the Jetpack-powered services selected earlier ?>
-				<?php esc_html_e( "Your store's almost ready. Connect to Jetpack for full access to Stripe payments, automated taxes, USPS live rates, and discounted shipping labels.", 'woocommerce' ); ?>
+				<?php echo $description ?>
```
Comment (mikejolley) [inline includes/admin/class-wc-admin-setup-wizard.php]: esc_html

File: includes/admin/class-wc-admin-setup-wizard.php [hunk 2]
```diff
@@ -1168,13 +1168,36 @@ class_exists( 'Jetpack' ) &&
...
 		$this->wc_setup_activate_actions();
+
+		$description = false;
+		$stripe_settings = get_option( 'woocommerce_stripe_settings', false );
+		$stripe_enabled = $stripe_settings && ! empty( $stripe_settings[ 'email' ] );
+		$taxes_enabled = get_option( 'woocommerce_setup_automated_taxes', false );
+		$rates_enabled = get_option( 'woocommerce_setup_domestic_live_rates_zone', false ) || get_option( 'woocommerce_setup_intl_live_rates_zone', false );
+
+		if ( $stripe_enabled && $taxes_enabled && $rates_enabled ) {
+			$description = __( "Your store's almost ready. Connect to Jetpack for full access to Stripe payments, automated taxes, live rates and discounted shipping labels.", 'woocommerce' );
```
Comment (mikejolley) [inline includes/admin/class-wc-admin-setup-wizard.php]: `store's` looks like bad english. Use `store is`.

**Review by mikejolley — CHANGES_REQUESTED** [review]
Small things to tweak.

**Comment by jeffstieler** [comment]
Issues addressed - cool to merge in @mikejolley? This is a pretty minor change.

**Review by jeffstieler — APPROVED** [review]
Approving these changes and merging. The logic for choosing the tailored text works, and I'd prefer to tweak copy all at once.

**Pull request merged**